src/components/Microsoft.DotNet.UpgradeAssistant/Checks/BypassableReadinessCheck.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Checks
```

Comment:
If this is something users should use, it should live in the Abstractions project.

Reply:
Removed